pkgs/development/tools/nsis/default.nix

```diff
@@ -28,30 +29,34 @@ stdenv.mkDerivation rec {
   '';
 
   nativeBuildInputs = [ sconsPackages.scons_3_1_2 ];
-  buildInputs = [ zlib ];
+  buildInputs = [ zlib libiconv ];
+
+  preBuild = (if stdenv.isLinux then ''
```

Comment:
You could use `lib.optionalString` here like so:
```suggestion
  preBuild = lib.optionalString stdenv.isLinux ''
```

Reply:
Thanks!